operators/tokenizer/clip_tokenizer.cc

```diff
@@ -177,14 +177,16 @@ void KernelClipBpeTokenizer::Compute(const ortc::Tensor<std::string>& input,
     }
   }
   int idx = 0;
+  // currently HF uses "<|endoftext|>" (which has id 49407) as default pad token
+  int pad_token = 49407;
```

Comment:
can this id value be queried by vocab? This hard code is not guaranteed if the vocab is changed.

Reply:
yep, done!